pkg/cmd/provisioning/identity_provider.go

```diff
@@ -110,7 +110,7 @@ func createIdentityProvider(client aws.Client, infraName, region, publicKeyPath,
 		Body:    awssdk.ReadSeekCloser(strings.NewReader(discoveryDocumentJSON)),
 		Bucket:  awssdk.String(bucketName),
 		Key:     awssdk.String(discoveryDocumentURI),
-		Tagging: awssdk.String(fmt.Sprintf("%s/%s=%s", ccoctlAWSResourceTagKeyPrefix, infraName, ownedCcoctlAWSResourceTagValue)),
+		Tagging: awssdk.String(fmt.Sprintf("%s/%s=%s", ccoctlAWSResourceTagKeyPrefix, namePrefix, ownedCcoctlAWSResourceTagValue)),
```

Comment:
adding nameprefix to the tag that's being added for ownership seems incorrect. this should be use for names only then?

Reply:
If we don't use some unique identifier, then all objects created by ccoctl will be tagged identically. How should we uniquely tag the objects?